app/models/user.rb

```diff
@@ -75,13 +75,40 @@ def tens
     end
   end
 
-  # def messages_count 
-  #   sum = 0
-  #   conversations = Conversation.where(
-  #         Conversation.where(conversations: {recipient_id: user.id})
-  #         .where(conversations: {sender_id: user.id})
-  #         .where_values.reduce(:or))
-  #   conversations.each do |c|
-  #     c.messages
+  def convo_count
+    Conversation.where(
+                Conversation.where(conversations: {recipient_id: self.id})
+                .where(conversations: {sender_id: self.id})
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Redundant `self` detected.<br>Space inside { missing.<br>Space inside } missing.